infra-as-code/bicep/modules/hubNetworking/parameters/hubNetworking.parameters.all.json

```diff
@@ -93,11 +96,14 @@
         "privatelink.cassandra.cosmos.azure.com",
         "privatelink.gremlin.cosmos.azure.com",
         "privatelink.table.cosmos.azure.com",
+        "privatelink.eastus.batch.azure.com",
```

Comment:
Why `eastus`? Do we need to add a comment or something saying this in this file incase they dont see the readme?